src/app/app.routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Routes } from '@angular/router';
+
+import { NotFoundPageComponent } from '@app/pages/core/not-found-page/not-found-page.component';
+
+import { AppRoutePath } from './enums/app-route-path.enum';
+
+export const appRoutes: Routes = [ {
+  path: '',
+  redirectTo: AppRoutePath.Courses,
+  pathMatch: 'full',
+}, {
+  path: '**',
+  component: NotFoundPageComponent,
+  data: {
+    showBreadcrumbs: false,
+  },
```

Comment:
It can be designed a bit simpler - just check for data presence and if here is no breadcrumbs data - do not show it

Reply:
Agree, will do it.